### PR TITLE
Add note on risk of split brains to zen.asciidoc

### PR DESCRIPTION
**Pull request by tback**
I propose this change to clarify documentation and to verify my
understanding of the implications of minimum_master_nodes and the
number of actual master nodes.

### CODE REVIEW

**Comment by tback** [comment]
I signed the CLA

**Review by jasontedor — COMMENTED** [review]
Thanks for the PR @tback. I left a comment, I'm not sure I see the need for this change?

File: docs/reference/modules/discovery/zen.asciidoc [hunk 1]
```diff
@@ -79,7 +79,8 @@ active master node will step down and a new master election will be begin.
 
 This setting must be set to a quorum of your master eligible nodes. It is recommended to avoid
 having only two master eligible nodes, since a quorum of two is two. Therefore, a loss
-of either master eligible node will result in an inoperable cluster.
+of either master eligible node will result in an inoperable cluster. It is also recommended to keep the 
+number of master nodes below `discovery.zen.minimum_master_nodes * 2` to avoid split brain scenarios.
```
Comment (jasontedor) [inline docs/reference/modules/discovery/zen.asciidoc]: We already say in this paragraph that "[t]his setting must be set to a quorum of your master eligible nodes", I think that's enough?

Comment (tback) [inline docs/reference/modules/discovery/zen.asciidoc]: No, to me that is not sufficient, I might not now what a _quorum_ implies. The documentation for the setting that I found just now is much more clear about this (https://www.elastic.co/guide/en/elasticsearch/reference/5.3/important-settings.html#minimum_master_nodes).

Maybe we can compromise on adding a link to the setting?

Comment (jasontedor) [inline docs/reference/modules/discovery/zen.asciidoc]: I would be good with providing a link from quorum that page that you reference. Do you want to make that change?

**Comment by tback** [comment]
I tried it with my last push, but I never wrote ascii doc before and right now I cannot check the outcome. It'd be great if you'd take over if this is broken - this took too much of your and my time already.

**Comment by jasontedor** [comment]
> It'd be great if you'd take over if this is broken - this took too much of your and my time already.

Sure, I'll push a change (we can link to the section that you reference via `<<minimum_master_nodes, master eligible nodes>>`). I'll still pull this is with you as the commit author assuming that you're okay with this?

**Comment by tback** [comment]
yeah, go ahead!

**Comment by jasontedor** [comment]
I pushed a98f93cb101a5d74a4bbd2189effcaa6d130ad8a, I'll pull this in. Thanks so much for starting the conversation here, I think this is an improvement!

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by jasontedor** [comment]
I pushed this to 5.3, 5.4, 5.x, and master.

**Comment by jasontedor** [comment]
Thanks @tback.

**Pull request merged**